bin/triton-bootstrap

```diff
@@ -253,8 +260,9 @@ else
         echo -n '.'
 
         curl -s -i -f -u $COUCHBASE_USER:$COUCHBASE_PASS \
-            "http://$CLUSTERIP:8091/controller/addNode" \
-            -d "hostname=$(ip addr show eth0 | grep -o '[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}')&user=admin&password=password"
+            -d "hostname=${IP_PRIVATE}&user=admin&password=password" \
```

Comment:
it looks like we're using `-u` to authenticate here. do we need these querystring params? And if so, should we swap them for the values in `$COUCHBASE_USER:$COUCHBASE_PASS`?